packages/generator/generator.js

```diff
@@ -790,12 +790,35 @@ async function generateRootReadme(fontDirectory) {
     variantCount += webfont.variants.length;
   }
 
-  let md = `# expo-google-fonts
+  let md = `<p align="center">
+  <a href="https://github.com/expo/google-fonts">
+    <img alt="Expo Google Fonts" src="./gifs/title.gif">
+  </a>
+</p>
+
+<p align="center">Use any of the 1052 fonts and variants from <a href="https://fonts.google.com" target="_blank">fonts.google.com</a> in your Expo app</p>
```

Comment:
The number here should be dynamically generated as below.
`${fontDirectory.items.length}`